CHANGELOG.md

```diff
@@ -129,7 +134,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed an issue where Alice would not verify if Bob's Bitcoin lock transaction is semantically correct, i.e. pays the agreed upon amount to an output owned by both of them.
   Fixing this required a **breaking change** on the network layer and hence old versions are not compatible with this version.
 
-[Unreleased]: https://github.com/comit-network/xmr-btc-swap/compare/0.7.0...HEAD
+[unreleased]: https://github.com/comit-network/xmr-btc-swap/compare/0.7.0...HEAD
```

Comment:
Why was this updated?

Reply:
Ughh I've been a victimized by Prettier this time :smile: (https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)

Reply:
We use [`dprint`](https://dprint.dev/) to format things in this repo. If you've ever been annoyed by how slow prettier is, give it a go :)